<!doctype html>
<html>
  <head>
    <title>Report None</title>
    <link
      href="https://cdn.datatables.net/2.1.4/css/dataTables.dataTables.min.css"
      rel="stylesheet"
    />

    <script src="https://code.jquery.com/jquery-3.7.0.min.js"></script>
    <script src="https://cdn.datatables.net/2.1.4/js/dataTables.min.js"></script>
    <script
      src="https://cdn.plot.ly/plotly-2.35.2.min.js"
      charset="utf-8"
    ></script>

    <style>
      body {
  font-family: Arial, sans-serif;
  font-size: small;
  margin: 0;
}

.sheet-outer {
  margin: 0;
}

.sheet {
  margin: 0;
  overflow: hidden;
  position: relative;
  box-sizing: border-box;
  page-break-after: always;
}

@media screen {
  body {
    background: #e0e0e0;
  }

  .sheet {
    background: white;
    box-shadow: 0 0.5mm 2mm rgba(0, 0, 0, 0.3);
    margin: 5mm auto;
  }
}

.sheet-outer.A4 .sheet {
  width: 210mm;
  height: 296mm;
}
.sheet.padding-5mm {
  padding: 5mm;
}

@page {
  size: A4;
  margin: 0;
}

@media print {
  .sheet-outer.A4,
  .sheet-outer.A5.landscape {
    width: 210mm;
    print-color-adjust: exact;
  }
}

.tb-cnv {
  border-top: 3px solid rgba(0, 0, 0, 0.4);
}

.tb-classification {
  border-top: 1px solid rgba(0, 0, 0, 0.4);
}

.tb-score th,
.tb-score td {
  width: 25%;
  vertical-align: top;
}

.tb-score td {
  position: relative;
  top: -85px;
  text-align: center;
  font-size: small;
}

.plot-container {
  display: flex;
  justify-content: center;
  align-items: center;
}

table {
  width: 100%;
  padding: 5px;
}

th {
  font-size: medium;
  font-weight: bold;
}

th,
td {
  text-align: left;
  padding: 10px;
}

table .tb-group {
  border-left: 1px solid black;
}

.tb-report td {
  vertical-align: top;
  padding: 0 25px 10px 25px;
  word-wrap: break-word;
}

.classification {
  width: 70%;
  text-align: center;
}

.divider {
  border-top: 1px solid gray;
  margin: 3px;
}

.tb-acmg,
.tb-genes,
.tb-elements {
  position: relative;
  top: -22mm;
}

.tb-acmg th {
  width: 50%;
  border-bottom: 1px solid rgba(0, 0, 0, 0.4);
  background-color: rgba(0, 0, 0, 0.1);
}

.tb-acmg td {
  padding: 0;
}

.tb-acmg-section,
.tb-acmg-section th,
.tb-acmg-section td {
  padding: 0;
}

h4 {
  margin-top: 0.67em;
  margin-bottom: 0.67em;
}

.tb-acmg-section p {
  margin-top: 0.67em;
  margin-bottom: 0.67em;
  font-size: smaller;
}

.td-score {
  width: 5%;
  text-align: center;
  vertical-align: bottom;
}

.score-box {
  border: 2px solid gray;
  border-radius: 5px;
  background-color: rgba(0, 0, 0, 0.1);
  padding: 8px 12px 8px 12px;
  font-size: larger;
  font-weight: bold;
}

.pathogenic {
  border: 2px solid rgb(225, 29, 29);
  background-color: rgba(225, 29, 29, 0.4);
}

.benign {
  border: 2px solid rgb(0, 128, 0);
  background-color: rgba(0, 128, 0, 0.4);
}

.scrollable {
  height: 120mm;
  overflow-y: scroll;
  border-right: 1px solid rgba(0, 0, 0, 0.4);
  padding: 0 10px 0 10px;
  /* -webkit-mask-image: linear-gradient(to bottom, black 90%, transparent 100%);
  mask-image: linear-gradient(to bottom, black 90%, transparent 100%); */
}

.last {
  border-right: none;
}

table.tb-genes,
table.tb-elements {
  /* border: 1px solid rgba(0, 0, 0, 0.4);*/
  border-radius: 5px;
}

.tb-genes th,
.tb-elements th {
  border-bottom: 1px solid rgba(0, 0, 0, 0.4);
  background-color: rgba(0, 0, 0, 0.1);
}

.tb-genes td,
.tb-elements td {
  border-right: 1px solid rgba(0, 0, 0, 0.4);
  width: 25%;
  padding: 10px;
  text-align: center;
  vertical-align: top;
}

.tb-genes td:first-child,
.tb-elements td:first-child {
  border-left: none;
}

.tb-genes td:last-child,
.tb-elements td:last-child {
  border-right: none;
}

.tb-elements td {
  width: 100%;
}

.list-scrollable {
  max-height: 40mm;
  overflow-y: scroll;
  /* -webkit-mask-image: linear-gradient(to bottom, black 70%, transparent 100%);
  mask-image: linear-gradient(to bottom, black 70%, transparent 100%); */
}

a {
  color: black;
  text-decoration: none;
}

a:hover {
  color: black;
  text-decoration: underline;
}

a:visited {
  color: gray;
}

    </style>
  </head>

  <body>
    <div class="sheet-outer A4">
      <section class="sheet padding-5mm">
        <h1 style="text-align: right">Report ID: None</h1>

        <table class="tb-cnv">
          <tbody>
  <tr>
    <th>Copy Number Variant LOSS</th>
    <th style="width: 60%; text-align: right">[GRCh38] 16q24.3(chr16:34454602-34454749)x1</th>
  </tr>
</tbody>
        </table>

        <table class="tb-classification">
          <tbody>
  <tr>
    <th>Automated classification:</th>
    
    <th
      class="classification"
      style="
        background-image: linear-gradient(
          to right,
          white,
          rgba(255, 165, 0, 0.6),
          white
        );
      "
    >
      LIKELY PATHOGENIC
    </th>
    
  </tr>
</tbody>
        </table>

        <table class="tb-score">
          <tbody>
  <tr>
    <th>Pathogenicity prediction:</th>
    <th>
      <div id="marcnv"></div>
    </th>
    <th>
      <div id="isv"></div>
    </th>
    <th>
      <div id="hybrid"></div>
    </th>
  </tr>

  <tr>
    <td></td>
    <td>marCNV score: 0.5</td>
    <td>ISV score: 1.0</td>
    <td>Hybrid ISV score: 0.75</td>
  </tr>
</tbody>
        </table>

        <table class="tb-acmg">
          <thead>
            <tr>
              <th>ACMG-ClinGen Standards (marCNV):</th>
              <th>ISV SHAP values:</th>
            </tr>
          </thead>

          <tbody>
            <tr>
              <td><div class="scrollable">
  <table class="tb-acmg-section">
    <tbody>
      <tr>
        <td>
          <h4>Section 1: Genomic content</h4>
          
          <p><b>1A.</b> Contains protein-coding or other known functionally important elements</p>
          
        </td>
        <td class="td-score">
          
          <div class="score-box pathogenic">0.1</div>
          
        </td>
      </tr>
    </tbody>
  </table>
  <div class="divider"></div>

  <table class="tb-acmg-section">
    <tbody>
      <tr>
        <td>
          <h4>Section 2: Overlap with established regions</h4>
          
          <p><b>2A.</b> Complete overlap; the TS gene or minimal critical region is fully contained within the observed copy number gain</p>
          
        </td>
        <td class="td-score">
          
          <div class="score-box pathogenic">1.0</div>
          
        </td>
      </tr>
    </tbody>
  </table>
  <div class="divider"></div>

  <table class="tb-acmg-section">
    <tbody>
      <tr>
        <td>
          <h4>Section 3: Gene numbers</h4>
          
          <p><b>3C.</b> 50 or more genes</p>
          
        </td>
        <td class="td-score">
          
          <div class="score-box pathogenic">0.9</div>
          
        </td>
      </tr>
    </tbody>
  </table>
  <div class="divider"></div>

  <table class="tb-acmg-section">
    <tbody>
      <tr>
        <td>
          <h4>Section 4: Public/internal data</h4>
          
          <p>No known fully overlapping population variant</p>
          
          <p>No Benign Gains fully overlapping (including LB)</p>
          
          <p>33 Pathogenic Gains fully contained (including LP)</p>
          
        </td>
        <td class="td-score">
          
          <div class="score-box pathogenic">0.5</div>
          
        </td>
      </tr>
    </tbody>
  </table>
  <div class="divider"></div>

  <table class="tb-acmg-section">
    <tbody>
      <tr>
        <td>
          <h4>Section 5: Inheritance pattern</h4>
          
          <p>No data available</p>
          
        </td>
        <td class="td-score">
          
          <div></div>
          
        </td>
      </tr>
    </tbody>
  </table>
</div></td>
              <td>
                <div class="last">
                  <div id="isv_shap"></div>
                </div>
              </td>
            </tr>
          </tbody>
        </table>

        <table class="tb-genes">
          <thead>
  <tr>
    <th colspan="4">Protein-coding genes: 106</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td>
      <h4>Morbid: 14</h4>
      <div class="list-scrollable">
        <p><a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=SLFN14" target=_blank>SLFN14</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=PEX12" target=_blank>PEX12</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNHIT3" target=_blank>ZNHIT3</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=PIGW" target=_blank>PIGW</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=HNF1B" target=_blank>HNF1B</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=GPR179" target=_blank>GPR179</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=PCGF2" target=_blank>PCGF2</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=NEUROD2" target=_blank>NEUROD2</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=TCAP" target=_blank>TCAP</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=EGFR" target=_blank>EGFR</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=MLH1" target=_blank>MLH1</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=MSH2" target=_blank>MSH2</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=MSH6" target=_blank>MSH6</a></p>
      </div>
    </td>
    <td>
      <h4>Disease-associated: 8</h4>
      <div class="list-scrollable">
        <p><a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=UNC45B" target=_blank>UNC45B</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=PEX12" target=_blank>PEX12</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ZNHIT3" target=_blank>ZNHIT3</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=PIGW" target=_blank>PIGW</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ACACA" target=_blank>ACACA</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=HNF1B" target=_blank>HNF1B</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=TCAP" target=_blank>TCAP</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ERBB2" target=_blank>ERBB2</a></p>
      </div>
    </td>
    <td>
      <h4>Haploinsufficient: 3</h4>
      <div class="list-scrollable">
        <p><a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ASIC2" target=_blank>ASIC2</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=CCL2" target=_blank>CCL2</a>, <a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=CCL7" target=_blank>CCL7</a></p>
      </div>
    </td>
    <td>
      <h4>Triplosensitive: 1</h4>
      <div class="list-scrollable">
        <p><a href="https://www.genecards.org/cgi-bin/carddisp.pl?gene=ASIC2" target=_blank>ASIC2</a></p>
      </div>
    </td>
  </tr>
</tbody>
        </table>

        <!--
        <table class="tb-elements">
          <thead>
            <tr>
              <th>Functionally important elements: elements.total</th>
            </tr>
          </thead>

          <tbody>
            <tr>
              <td>
                <div class="list-scrollable">
                  elements.list
                </div>
              </td>
            </tr>
          </tbody>
        </table>
    -->
      </section>
    </div>

    <script>
      // Function to set the color of the portion of the pie chart and set the other portions to invisible
      function setColor(pie, score) {
        var trace = {
          text: ["B", "LB", "VUS", "LP", "P", " "],
          marker: {
            colors: [
              "rgba(154, 205, 50, 0.3)",
              "rgba(144, 238, 144, 0.3)",
              "rgba(255, 255, 0, 0.3)",
              "rgba(255, 165, 0, 0.3)",
              "rgba(255, 0, 0, 0.3)",
              "white",
            ],
            line: {
              color: [
                "rgba(154, 205, 50, 1)",
                "rgba(144, 238, 144, 1)",
                "rgba(255, 230, 0, 1)",
                "rgba(255, 165, 0, 1)",
                "rgba(255, 0, 0, 1)",
                "white",
              ],
              width: [0, 0, 0, 0, 0, 0],
            },
          },
          pull: [0, 0, 0, 0, 0, 0],
        };

        switch (pie) {
          case "marcnv":
            if (score <= -0.9) {
              trace.marker.colors[0] = "rgba(154, 205, 50, 0.6)";
              trace.marker.line.color[0] = "rgba(154, 205, 50, 1)";
              trace.marker.line.width[0] = 3;
              trace.pull[0] = 0.1;
              trace.text[0] = "<b>B</b>";
            } else if (score > -0.9 && score < -0.5) {
              trace.marker.colors[1] = "rgba(144, 238, 144, 0.6)";
              trace.marker.line.color[1] = "rgba(144, 238, 144, 1)";
              trace.marker.line.width[1] = 3;
              trace.pull[1] = 0.1;
              trace.text[1] = "<b>LB</b>";
            } else if (score >= -0.5 && score <= 0.5) {
              trace.marker.colors[2] = "rgba(255, 230, 0, 0.6)";
              trace.marker.line.color[2] = "rgba(255, 230, 0, 1)";
              trace.marker.line.width[2] = 3;
              trace.pull[2] = 0.1;
              trace.text[2] = "<b>VUS</b>";
            } else if (score > 0.5 && score < 0.9) {
              trace.marker.colors[3] = "rgba(255, 165, 0, 0.6)";
              trace.marker.line.color[3] = "rgba(255, 165, 0, 1)";
              trace.marker.line.width[3] = 3;
              trace.pull[3] = 0.1;
              trace.text[3] = "<b>LP</b>";
            } else if (score >= 0.9) {
              trace.marker.colors[4] = "rgba(255, 0, 0, 0.6)";
              trace.marker.line.color[4] = "rgba(255, 0, 0, 1)";
              trace.marker.line.width[4] = 3;
              trace.pull[4] = 0.1;
              trace.text[4] = "<b>P</b>";
            }
            break;
          case "isv":
            if (score <= 0.1) {
              trace.marker.colors[0] = "rgba(154, 205, 50, 0.6)";
              trace.marker.line.color[0] = "rgba(154, 205, 50, 1)";
              trace.marker.line.width[0] = 3;
              trace.pull[0] = 0.1;
              trace.text[0] = "<b>B</b>";
            } else if (score > 0.1 && score < 0.25) {
              trace.marker.colors[1] = "rgba(144, 238, 144, 0.6)";
              trace.marker.line.color[1] = "rgba(144, 238, 144, 1)";
              trace.marker.line.width[1] = 3;
              trace.pull[1] = 0.1;
              trace.text[1] = "<b>LB</b>";
            } else if (score >= 0.25 && score <= 0.75) {
              trace.marker.colors[2] = "rgba(255, 230, 0, 0.6)";
              trace.marker.line.color[2] = "rgba(255, 230, 0, 1)";
              trace.marker.line.width[2] = 3;
              trace.pull[2] = 0.1;
              trace.text[2] = "<b>VUS</b>";
            } else if (score > 0.75 && score < 0.9) {
              trace.marker.colors[3] = "rgba(255, 165, 0, 0.6)";
              trace.marker.line.color[3] = "rgba(255, 165, 0, 1)";
              trace.marker.line.width[3] = 3;
              trace.pull[3] = 0.1;
              trace.text[3] = "<b>LP</b>";
            } else if (score >= 0.9) {
              trace.marker.colors[4] = "rgba(255, 0, 0, 0.6)";
              trace.marker.line.color[4] = "rgba(255, 0, 0, 1)";
              trace.marker.line.width[4] = 3;
              trace.pull[4] = 0.1;
              trace.text[4] = "<b>P</b>";
            }
            break;
          case "hybrid":
            if (score <= 0.2) {
              trace.marker.colors[0] = "rgba(154, 205, 50, 0.6)";
              trace.marker.line.color[0] = "rgba(154, 205, 50, 1)";
              trace.marker.line.width[0] = 3;
              trace.pull[0] = 0.1;
              trace.text[0] = "<b>B</b>";
            } else if (score > 0.2 && score < 0.4) {
              trace.marker.colors[1] = "rgba(144, 238, 144, 0.6)";
              trace.marker.line.color[1] = "rgba(144, 238, 144, 1)";
              trace.marker.line.width[1] = 3;
              trace.pull[1] = 0.1;
              trace.text[1] = "<b>LB</b>";
            } else if (score >= 0.4 && score <= 0.6) {
              trace.marker.colors[2] = "rgba(255, 230, 0, 0.6)";
              trace.marker.line.color[2] = "rgba(255, 230, 0, 1)";
              trace.marker.line.width[2] = 3;
              trace.pull[2] = 0.1;
              trace.text[2] = "<b>VUS</b>";
            } else if (score > 0.6 && score < 0.8) {
              trace.marker.colors[3] = "rgba(255, 165, 0, 0.6)";
              trace.marker.line.color[3] = "rgba(255, 165, 0, 1)";
              trace.marker.line.width[3] = 3;
              trace.pull[3] = 0.1;
              trace.text[3] = "<b>LP</b>";
            } else if (score >= 0.8) {
              trace.marker.colors[4] = "rgba(255, 0, 0, 0.6)";
              trace.marker.line.color[4] = "rgba(255, 0, 0, 1)";
              trace.marker.line.width[4] = 3;
              trace.pull[4] = 0.1;
              trace.text[4] = "<b>P</b>";
            }
            break;
        }

        return trace;
      }

      var trace = {
        type: "pie",
        showlegend: false,
        hole: 0.4,
        rotation: 90,
        values: [100 / 5, 100 / 5, 100 / 5, 100 / 5, 100 / 5, 100],
        direction: "clockwise",
        textinfo: "text",
        textposition: "inside",
        labels: [
          "Benign",
          "Likely Benign",
          "Uncertain Significance",
          "Likely Pathogenic",
          "Pathogenic",
          " ",
        ],
        hoverinfo: "label",
      };

      const traceMarCNV = Object.assign(
        { ...trace },
        setColor("marcnv", parseFloat("0.5")),
      );

      const traceISV = Object.assign(
        { ...trace },
        setColor("isv", parseFloat("1.0")),
      );

      const traceHybrid = Object.assign(
        { ...trace },
        setColor("hybrid", parseFloat("0.75")),
      );

      var degrees = 115,
        radius = 0.6;
      var radians = (degrees * Math.PI) / 180;
      var x = -1 * radius * Math.cos(radians);
      var y = radius * Math.sin(radians);

      var layout = {
        width: 150,
        height: 150,
        margin: { t: 1, b: 1, l: 1, r: 1 },
        xaxis: { visible: false, range: [-1, 1] },
        yaxis: { visible: false, range: [-1, 1] },
      };

      const config = {
        displayModeBar: false,
      };

      Plotly.newPlot("marcnv", [traceMarCNV], layout, config);
      Plotly.newPlot("isv", [traceISV], layout, config);
      Plotly.newPlot("hybrid", [traceHybrid], layout, config);

      var isv_shap = {"data":[{"hoverinfo":"text","hovertext":["\u003cb\u003e\u003ci\u003eTriplosensitivity Genes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 1\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.0","\u003cb\u003e\u003ci\u003eTriplosensitive Regions\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 1\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.02","\u003cb\u003e\u003ci\u003eTranscriptional cis-regulatory Regions\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 12\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.1","\u003cb\u003e\u003ci\u003eTATA box\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 0\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.04","\u003cb\u003e\u003ci\u003eSmall nuclear RNA\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 4\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.12","\u003cb\u003e\u003ci\u003eSilencers\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 33\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.01","\u003cb\u003e\u003ci\u003eRibosomal RNA\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 18\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.02","\u003cb\u003e\u003ci\u003eRegulatory Elements\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 278\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.01","\u003cb\u003e\u003ci\u003ePseudogenes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 56\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.25","\u003cb\u003e\u003ci\u003eProtein Coding Genes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 106\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.03","\u003cb\u003e\u003ci\u003ePromoters\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 46\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.02","\u003cb\u003e\u003ci\u003eOverlapped Gencode Elements\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 205\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.02","\u003cb\u003e\u003ci\u003eMorbid Genes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 14\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.3","\u003cb\u003e\u003ci\u003eMicro RNA\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 2\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.01","\u003cb\u003e\u003ci\u003eLong non-coding RNA\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 5\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.07","\u003cb\u003e\u003ci\u003eHaploinsufficient Regions\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 17\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.001","\u003cb\u003e\u003ci\u003eHaploinsufficient Genes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 3\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.22","\u003cb\u003e\u003ci\u003eEnhancers\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 65\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.02","\u003cb\u003e\u003ci\u003eEnhancer-blocking Elements\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 8\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e -0.03","\u003cb\u003e\u003ci\u003eDisease associated Genes\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 8\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.45","\u003cb\u003e\u003ci\u003eDNase I hypersensitive sites\u003c\u002fi\u003e\u003c\u002fb\u003e\u003cbr\u003e\u003cb\u003evalue:\u003c\u002fb\u003e 2\u003cbr\u003e\u003cb\u003eSHAP value:\u003c\u002fb\u003e 0.05"],"marker":{"color":["gray","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(225, 29, 29)","rgb(225, 29, 29)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(0, 128, 0)","rgb(225, 29, 29)","rgb(225, 29, 29)"]},"orientation":"h","text":["0.0","0.02","-0.1","0.04","0.12","0.01","0.02","-0.01","0.25","-0.03","0.02","0.02","-0.3","0.01","-0.07","0.001","-0.22","0.02","-0.03","0.45","0.05"],"textfont":{"size":7},"textposition":"outside","x":[0.0,0.02,-0.1,0.04,0.12,0.01,0.02,-0.01,0.25,-0.03,0.02,0.02,-0.3,0.01,-0.07,0.001,-0.22,0.02,-0.03,0.45,0.05],"y":["[1.0] = Triplosensitivity Genes","[1.0] = Triplosensitive Regions","[12.0] = Transcriptional cis-regulatory Regions","[0.0] = TATA box","[4.0] = Small nuclear RNA","[33.0] = Silencers","[18.0] = Ribosomal RNA","[278.0] = Regulatory Elements","[56.0] = Pseudogenes","[106.0] = Protein Coding Genes","[46.0] = Promoters","[205.0] = Overlapped Gencode Elements","[14.0] = Morbid Genes","[2.0] = Micro RNA","[5.0] = Long non-coding RNA","[17.0] = Haploinsufficient Regions","[3.0] = Haploinsufficient Genes","[65.0] = Enhancers","[8.0] = Enhancer-blocking Elements","[8.0] = Disease associated Genes","[2.0] = DNase I hypersensitive sites"],"type":"bar"}],"layout":{"template":{"data":{"barpolar":[{"marker":{"line":{"color":"white","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"barpolar"}],"bar":[{"error_x":{"color":"#2a3f5f"},"error_y":{"color":"#2a3f5f"},"marker":{"line":{"color":"white","width":0.5},"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"bar"}],"carpet":[{"aaxis":{"endlinecolor":"#2a3f5f","gridcolor":"#C8D4E3","linecolor":"#C8D4E3","minorgridcolor":"#C8D4E3","startlinecolor":"#2a3f5f"},"baxis":{"endlinecolor":"#2a3f5f","gridcolor":"#C8D4E3","linecolor":"#C8D4E3","minorgridcolor":"#C8D4E3","startlinecolor":"#2a3f5f"},"type":"carpet"}],"choropleth":[{"colorbar":{"outlinewidth":0,"ticks":""},"type":"choropleth"}],"contourcarpet":[{"colorbar":{"outlinewidth":0,"ticks":""},"type":"contourcarpet"}],"contour":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"contour"}],"heatmapgl":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"heatmapgl"}],"heatmap":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"heatmap"}],"histogram2dcontour":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"histogram2dcontour"}],"histogram2d":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"histogram2d"}],"histogram":[{"marker":{"pattern":{"fillmode":"overlay","size":10,"solidity":0.2}},"type":"histogram"}],"mesh3d":[{"colorbar":{"outlinewidth":0,"ticks":""},"type":"mesh3d"}],"parcoords":[{"line":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"parcoords"}],"pie":[{"automargin":true,"type":"pie"}],"scatter3d":[{"line":{"colorbar":{"outlinewidth":0,"ticks":""}},"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scatter3d"}],"scattercarpet":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scattercarpet"}],"scattergeo":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scattergeo"}],"scattergl":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scattergl"}],"scattermapbox":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scattermapbox"}],"scatterpolargl":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scatterpolargl"}],"scatterpolar":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scatterpolar"}],"scatter":[{"fillpattern":{"fillmode":"overlay","size":10,"solidity":0.2},"type":"scatter"}],"scatterternary":[{"marker":{"colorbar":{"outlinewidth":0,"ticks":""}},"type":"scatterternary"}],"surface":[{"colorbar":{"outlinewidth":0,"ticks":""},"colorscale":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"type":"surface"}],"table":[{"cells":{"fill":{"color":"#EBF0F8"},"line":{"color":"white"}},"header":{"fill":{"color":"#C8D4E3"},"line":{"color":"white"}},"type":"table"}]},"layout":{"annotationdefaults":{"arrowcolor":"#2a3f5f","arrowhead":0,"arrowwidth":1},"autotypenumbers":"strict","coloraxis":{"colorbar":{"outlinewidth":0,"ticks":""}},"colorscale":{"diverging":[[0,"#8e0152"],[0.1,"#c51b7d"],[0.2,"#de77ae"],[0.3,"#f1b6da"],[0.4,"#fde0ef"],[0.5,"#f7f7f7"],[0.6,"#e6f5d0"],[0.7,"#b8e186"],[0.8,"#7fbc41"],[0.9,"#4d9221"],[1,"#276419"]],"sequential":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]],"sequentialminus":[[0.0,"#0d0887"],[0.1111111111111111,"#46039f"],[0.2222222222222222,"#7201a8"],[0.3333333333333333,"#9c179e"],[0.4444444444444444,"#bd3786"],[0.5555555555555556,"#d8576b"],[0.6666666666666666,"#ed7953"],[0.7777777777777778,"#fb9f3a"],[0.8888888888888888,"#fdca26"],[1.0,"#f0f921"]]},"colorway":["#636efa","#EF553B","#00cc96","#ab63fa","#FFA15A","#19d3f3","#FF6692","#B6E880","#FF97FF","#FECB52"],"font":{"color":"#2a3f5f"},"geo":{"bgcolor":"white","lakecolor":"white","landcolor":"white","showlakes":true,"showland":true,"subunitcolor":"#C8D4E3"},"hoverlabel":{"align":"left"},"hovermode":"closest","mapbox":{"style":"light"},"paper_bgcolor":"white","plot_bgcolor":"white","polar":{"angularaxis":{"gridcolor":"#EBF0F8","linecolor":"#EBF0F8","ticks":""},"bgcolor":"white","radialaxis":{"gridcolor":"#EBF0F8","linecolor":"#EBF0F8","ticks":""}},"scene":{"xaxis":{"backgroundcolor":"white","gridcolor":"#DFE8F3","gridwidth":2,"linecolor":"#EBF0F8","showbackground":true,"ticks":"","zerolinecolor":"#EBF0F8"},"yaxis":{"backgroundcolor":"white","gridcolor":"#DFE8F3","gridwidth":2,"linecolor":"#EBF0F8","showbackground":true,"ticks":"","zerolinecolor":"#EBF0F8"},"zaxis":{"backgroundcolor":"white","gridcolor":"#DFE8F3","gridwidth":2,"linecolor":"#EBF0F8","showbackground":true,"ticks":"","zerolinecolor":"#EBF0F8"}},"shapedefaults":{"line":{"color":"#2a3f5f"}},"ternary":{"aaxis":{"gridcolor":"#DFE8F3","linecolor":"#A2B1C6","ticks":""},"baxis":{"gridcolor":"#DFE8F3","linecolor":"#A2B1C6","ticks":""},"bgcolor":"white","caxis":{"gridcolor":"#DFE8F3","linecolor":"#A2B1C6","ticks":""}},"title":{"x":0.05},"xaxis":{"automargin":true,"gridcolor":"#EBF0F8","linecolor":"#EBF0F8","ticks":"","title":{"standoff":15},"zerolinecolor":"#EBF0F8","zerolinewidth":2},"yaxis":{"automargin":true,"gridcolor":"#EBF0F8","linecolor":"#EBF0F8","ticks":"","title":{"standoff":15},"zerolinecolor":"#EBF0F8","zerolinewidth":2}}},"shapes":[{"line":{"color":"black","width":1},"type":"line","x0":0,"x1":0,"xref":"x","y0":0,"y1":1,"yref":"y domain"}],"xaxis":{"tickfont":{"size":7},"range":[-1,1]},"yaxis":{"tickfont":{"size":7},"tickmode":"array","tickvals":["[1.0] = Triplosensitivity Genes","[1.0] = Triplosensitive Regions","[12.0] = Transcriptional cis-regulatory Regions","[0.0] = TATA box","[4.0] = Small nuclear RNA","[33.0] = Silencers","[18.0] = Ribosomal RNA","[278.0] = Regulatory Elements","[56.0] = Pseudogenes","[106.0] = Protein Coding Genes","[46.0] = Promoters","[205.0] = Overlapped Gencode Elements","[14.0] = Morbid Genes","[2.0] = Micro RNA","[5.0] = Long non-coding RNA","[17.0] = Haploinsufficient Regions","[3.0] = Haploinsufficient Genes","[65.0] = Enhancers","[8.0] = Enhancer-blocking Elements","[8.0] = Disease associated Genes","[2.0] = DNase I hypersensitive sites"],"ticktext":["\u003cspan style=\"color: gray;\"\u003e[1.0] = \u003c\u002fspan\u003eTriplosensitivity Genes","\u003cspan style=\"color: gray;\"\u003e[1.0] = \u003c\u002fspan\u003eTriplosensitive Regions","\u003cspan style=\"color: gray;\"\u003e[12.0] = \u003c\u002fspan\u003eTranscriptional cis-regulatory Regions","\u003cspan style=\"color: gray;\"\u003e[0.0] = \u003c\u002fspan\u003eTATA box","\u003cspan style=\"color: gray;\"\u003e[4.0] = \u003c\u002fspan\u003eSmall nuclear RNA","\u003cspan style=\"color: gray;\"\u003e[33.0] = \u003c\u002fspan\u003eSilencers","\u003cspan style=\"color: gray;\"\u003e[18.0] = \u003c\u002fspan\u003eRibosomal RNA","\u003cspan style=\"color: gray;\"\u003e[278.0] = \u003c\u002fspan\u003eRegulatory Elements","\u003cspan style=\"color: gray;\"\u003e[56.0] = \u003c\u002fspan\u003ePseudogenes","\u003cspan style=\"color: gray;\"\u003e[106.0] = \u003c\u002fspan\u003eProtein Coding Genes","\u003cspan style=\"color: gray;\"\u003e[46.0] = \u003c\u002fspan\u003ePromoters","\u003cspan style=\"color: gray;\"\u003e[205.0] = \u003c\u002fspan\u003eOverlapped Gencode Elements","\u003cspan style=\"color: gray;\"\u003e[14.0] = \u003c\u002fspan\u003eMorbid Genes","\u003cspan style=\"color: gray;\"\u003e[2.0] = \u003c\u002fspan\u003eMicro RNA","\u003cspan style=\"color: gray;\"\u003e[5.0] = \u003c\u002fspan\u003eLong non-coding RNA","\u003cspan style=\"color: gray;\"\u003e[17.0] = \u003c\u002fspan\u003eHaploinsufficient Regions","\u003cspan style=\"color: gray;\"\u003e[3.0] = \u003c\u002fspan\u003eHaploinsufficient Genes","\u003cspan style=\"color: gray;\"\u003e[65.0] = \u003c\u002fspan\u003eEnhancers","\u003cspan style=\"color: gray;\"\u003e[8.0] = \u003c\u002fspan\u003eEnhancer-blocking Elements","\u003cspan style=\"color: gray;\"\u003e[8.0] = \u003c\u002fspan\u003eDisease associated Genes","\u003cspan style=\"color: gray;\"\u003e[2.0] = \u003c\u002fspan\u003eDNase I hypersensitive sites"]},"margin":{"l":20,"r":20,"t":20,"b":20},"height":450,"width":400}};
      Plotly.react("isv_shap", isv_shap);
    </script>
  </body>
</html>